### PR TITLE
fixed readme

### PR DESCRIPTION
**Pull request by jessica-tw**
Signed-off-by: jessica-tw <jessica.olinda@zup.com.br>

- Updated README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+2. [**Getting Started**](#getting-started)
+> #### i. [**Requirements**](#requirements)
+> #### ii. [**Installation**](#installation)
+> #### iii. [**Build**](#build)
+3. [**How to contribute?**](#how-to-contribute)
+4. [**Code of Conduct**](#code-of-conduct)
+5. [**Community**](#community)
+
+## **About** 
 ### All your automations in one place
```
Comment (mdcpmoreira) [inline README.md:15]: I think that we don't need to maintain this introduction, considering we want to establish the same pattern for all docs repositories. I'd just delete these phrase and go straight forward "Ritchie CLI is an open source..."

File: README.md [hunk 2]
```diff
@@ -1,25 +1,37 @@
...
 
 **Ritchie CLI** is an open source project that allows to **create**, **store** and **share** automations, executing them through command lines.
```
Comment (mdcpmoreira) [inline README.md:17]:
```suggestion
**Ritchie CLI** is an open source project that allows to create, store and share automations, executing them through command lines.
```

File: README.md [hunk 3]
```diff
@@ -55,22 +67,25 @@ npm start
 
 - Now, check if the `Web Server is available at http://localhost:1313/docs/`.
 
-## **Running the website locally**
+### **Running the website locally**
 
 Once you've cloned this repo, from the `root` folder, run:
```
Comment (mdcpmoreira) [inline README.md:72]:
```suggestion
Once you've cloned this repo, from the `root` folder, run the command:
```

File: README.md [hunk 4]
```diff
@@ -55,22 +67,25 @@ npm start
...
 
 ```
 hugo server
 ```
 
-# **How to contribute?**
+## **How to contribute?**
 
 You can suggest a change, a fix, or improvements to our documentation, for example:
```
Comment (mdcpmoreira) [inline README.md:80]:
```suggestion
You can suggest a change, a fix, or improvements to our documentation, such as: 
```

**Review by mdcpmoreira — COMMENTED** [review]
TW review done ;)

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,30 +1,100 @@
...
+![git-branchs](https://user-images.githubusercontent.com/22433243/117700179-0f060780-b19c-11eb-8d03-42cfbd2de798.png)
+
+**Examples:** `feature/name` or `fix/name`
+
+**Step 3.** Make your changes;
+
+**Step 4.** Open a pull request open a PR to the published branch, i. e., **`release-VERSION`** and fill in the template; 
+
+**Step 5.** CharlesCD's team will check your issue, review it, and then approve your PR. 
+Is this the first CharlesCD repository you visit? Check out other [**repositories**](https://github.com/ZupIT/ritchie-cli).
```
Comment (lucasdittrichzup) [inline CONTRIBUTING.md:90]: Ritchie CLI

File: README.md [hunk 6]
```diff
@@ -1,25 +1,36 @@
 # **Ritchie CLI Documentation**
-This repository uses [**hugo server**](https://gohugo.io/commands/hugo_server/) and contains the [**Ritchie CLI**](https://ritchiecli.io/) official documentation. 
+This repository uses [**hugo server**](https://gohugo.io/commands/hugo_server/) and contains [**Ritchie CLI**](https://ritchiecli.io/) official documentation.
```
Comment (lucasdittrichzup) [inline README.md:2]: the correct link for the doc is: https://docs.ritchiecli.io/

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,30 +1,100 @@
...
-information on using pull requests.
+### **Community Guidelines**
 
-## **Community Guidelines**
+This project follows [**Google's Open Source Community Guidelines**](https://opensource.google.com/conduct/).
 
-This project follows
-[**Google's Open Source Community Guidelines**](https://opensource.google.com/conduct/).
+## **Prerequisites**
+Check out the requisites before contributing to CharlesCD documentation:
```
Comment (lucasdittrichzup) [inline CONTRIBUTING.md:26]: Ritchie CLI

**Review by lucasdittrichzup — CHANGES_REQUESTED** [review]

**Review by lucasdittrichzup — APPROVED** [review]

**Review by GuillaumeFalourd — APPROVED** [review]
👍🏼

**Pull request merged**